modules/internal/DefaultRectangular.chpl

```diff
@@ -868,9 +868,16 @@ module DefaultRectangular {
     }
   
     // only need second version because wrapper record can pass a 1-tuple
-    inline proc dsiAccess(ind: idxType ...1) ref where rank == 1
+    inline proc dsiAccess(ind: idxType ...1) ref
+    where rank == 1
```

Comment:
These three variants are visually hard to separate.
Let's put 'where' clauses back on the respective 'proc' lines.